src/rpm_ostree/cli_status.rs

```diff
@@ -85,37 +85,62 @@ impl Deployment {
             checksum: self.base_revision(),
             version: self.version,
             age_index: None,
+            is_oci: self.container_image_reference.is_some(),
         }
     }
 
     /// Return the deployment base revision.
     pub fn base_revision(&self) -> String {
-        self.base_checksum
+        self.container_image_reference
             .clone()
+            .map(|pullspec| {
+                pullspec
+                    .strip_prefix("ostree-remote-image:fedora:docker://")
+                    .map(|s| s.to_string())
+            })
+            .flatten()
+            .or(self.base_checksum.clone())
```

Comment:
Maybe it would make more sense to add an `Option<String>` to the `Deployement` struct to be able to have both the pullspec and the base commit ? 
However the cincinnati graph does not contains checksum in the OCI case so we can't compare ostree commits when doing `rebase` later anyway.

Reply:
haven't had the need for it in the Zincati context